test/classes/deinitializers/deinitAndDestruct.good

```diff
@@ -0,0 +1,3 @@
+deinitAndDestruct.chpl:11: error: ambiguous call 'C.deinit()'
+deinitAndDestruct.chpl:2: note: candidates are: C.~C()
+deinitAndDestruct.chpl:6: note:                 C.deinit()
```

Comment:
Do you anticipate that being a huge implementation effort?  The check was relatively straight-forward for initializers